lib/phoenix/transports/websocket.ex

```diff
@@ -22,7 +22,13 @@ defmodule Phoenix.Transports.WebSocket do
       origin header is present. It defaults to true and, in such cases,
       it will check against the host value in `YourApp.Endpoint.config(:url)[:host]`.
       It may be set to `false` (not recommended) or to a list of explicitly
-      allowed origins
+      allowed origins.
+
+      check_origin: ["https://example.com",
+                     "//another.com:888", "//other.com"]
+
+      Note: it is necessary to set `check_origin: false` to support native
```

Comment:
Ah, I was going off of https://github.com/davidstump/SwiftPhoenixClient/issues/21#issuecomment-157050293 which is quite likely outdated by this point.

Updated the wording.